### PR TITLE
expand documentation for the <logfile> component

### PR DESCRIPTION
**Pull request by freddrake**
Covers all options for ``<logfile>`` handlers.

### CODE REVIEW

File: ZConfig/components/logger/handlers.xml [hunk 1]
```diff
@@ -12,46 +12,126 @@
...
     <key name="format"
          default="------\n%(asctime)s %(levelname)s %(name)s %(message)s"
-         datatype=".log_format"/>
+         datatype=".log_format">
+      <description format="rest">
+        Format string for log entries.  This value is used to create an
+        instance of the class identified by ``formatter``.
+
+        The following escape characters are supported with the same
+        replacements as in Python string literals:
```
Comment (jamadden) [inline ZConfig/components/logger/handlers.xml]: Is it two colons `::` for a pre-formatted block? I thought simple indentation was a quoted block.

File: ZConfig/components/logger/handlers.xml [hunk 2]
```diff
@@ -12,46 +12,126 @@
...
+
+        This is used with Python's :func:`time.strftime` function,
+        so must be compatible with that function on the host platform.
+      </description>
+    </key>
     <key name="level"
          default="notset"
-         datatype="ZConfig.components.logger.datatypes.logging_level"/>
+         datatype="ZConfig.components.logger.datatypes.logging_level">
+      <description format="rest">
```
Comment (jamadden) [inline ZConfig/components/logger/handlers.xml]: The `format` attribute seems out of place (I don't see it anywhere else).

Comment (freddrake) [reply ZConfig/components/logger/handlers.xml]: Indeed, it appears to be unnecessary.

Changed in commit 5f85187.

File: ZConfig/components/logger/handlers.xml (hunk 1 above)
Comment (freddrake) [reply ZConfig/components/logger/handlers.xml]: Thanks for catching this!  I clearly reviewed the formatted HTML too quickly.

Fixed in commit 5f85187.

**Pull request merged**